### DISCUSSION TITLE
Switch language on the fly

### DISCUSSION THREAD

**Discussion by ergodron** [surmon-china/vue-codemirror#163] [Q&A]
### Clear and concise description of the problem

Could you suggest how to implement language switching on the fly?

### Suggested solution

Couldn't find it in the documentation

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by surmon-china** [comment]
In CodeMirror@6, the language is just an extension and you only need to control the data of the extensions dynamically, just like the [example site](https://github.com/surmon-china/surmon-china.github.io/blob/source/examples/vue-codemirror/editor.vue#L79-L88) does.
